core/src/main/scala/caliban/validation/Validator.scala

```diff
@@ -161,7 +162,7 @@ object Validator {
     IO.foreach(selectionSet) {
         case FragmentSpread(_, directives) =>
           checkDirectivesUniqueness(directives).as(directives.map((_, __DirectiveLocation.FRAGMENT_SPREAD)))
-        case Field(_, _, _, directives, selectionSet) =>
+        case Field(_, _, _, directives, selectionSet, _) =>
```

Comment:
since we don't match on special values, again just `case f: Field => ...` access f members where needed